### PR TITLE
Custom Page titles with extra static part

### PR DESCRIPTION
**Pull request by TomSchillemans**

### CODE REVIEW

**Comment by bcjordan** [comment]
Nice!

File: rails/display-custom-page-title-with-extra.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+It is an easy one but I think it is an helpful TIL. Especially since the static part of the website allows 
+users searching on your site to see what your site is about!
+
+
+```haml
+%title= (yield(:title) + ' :: ' unless yield(:title).blank?).to_s + ' Makichan - Anime pinboard'
+
+```
+
+This is snipped directly copied from my template file. The ``` '::' ``` part is the delimiter,
```
Comment (bcjordan) [inline rails/display-custom-page-title-with-extra.md]: > snipped directly copied from

Should this be either `snipped directly` / `copied from` versus the pair?

File: rails/display-custom-page-title-with-extra.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+```haml
+%title= (yield(:title) + ' :: ' unless yield(:title).blank?).to_s + ' Makichan - Anime pinboard'
+
+```
+
+This is snipped directly copied from my template file. The ``` '::' ``` part is the delimiter, 
+to be shows between the "dynamic" and "static" parts of the title. The second string is the static part of the title,
+that part is always shown.
```
Comment (bcjordan) [inline rails/display-custom-page-title-with-extra.md]: > [...] that part is always shown

could potentially be

> [...] which is always shown

File: rails/display-custom-page-title-with-extra.md (hunk 1 above)
Comment (TomSchillemans) [inline rails/display-custom-page-title-with-extra.md]: sorry yeah that is an awful mistake on my part :-1:

**Comment by TomSchillemans** [comment]
Also, should I add some visual examples?

**Comment by stefannibrasil** [comment]
This repo is no longer active and it's being archived. The contents will still be available as read-only. Thank you for your contributions!

**Pull request closed without merging**